MockDuck/Sources/RequestResponseCommonProtocol.swift

```diff
@@ -25,6 +25,12 @@ extension RequestResponseCommonProtocol {
             return "gif"
         } else if contentType.contains("application/json") {
             return "json"
+        } else if contentType.contains("application/x-www-form-urlencoded") {
+            return "txt"
+        } else if contentType.hasPrefix("text/") {
```

Comment:
What are some examples that this case will catch?

Reply:
text/html, text/css, text/javascript